natiami/assignment3/FindWordsTest.h

```diff
@@ -0,0 +1,16 @@
+#include<string>
+#include<vector>
+#include<set>
+#include"Grid.h"
+#include"lib.h"
+
+class FindWordsTest{
+public:
+	void test();
+private:
+	Grid* initGrid();
+	Dictionary* initDictionary();
+	void test_findAllWords();
+	void test_EmptyGrid();
+	void test_EmptyDictionary();
+};
```

Comment:
That's a great test class structure and implementation! 👍 
Tests are separated to functions, and from the function name it is clear what is being tested.